### PR TITLE
enforce-signing: Add section with an enforce signing policy

### PR DESCRIPTION
**Pull request by hectorj2f**
## Type of change

Add an example of a policy
### What should this PR do?

Shows to users how to create policies for their enforce  signed images
### Why are we making this change?

We don't have any specific documentation about this scenario.

### CODE REVIEW

**Comment by hectorj2f** [comment]
@ltagliaferri As discussed on slack, I am proposing some documentation to show an example of a policy targeting image signed via enforce.

File: content/chainguard/chainguard-enforce/chainguard-enforce-signing/verify-enforce-signed-images.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  images:
+  - glob: image/*
+  authorities:
+  - keyless:
+      trustRootRef: my-sigstore-keys
+      url: https://<my-enforce-sigstore-hostname-instance>
+      identities:
+        - issuer: 'https://auth.chainguard.dev/'
+          subject: 'foo@example.com'
+      insecureIgnoreSCT: true # In Sigaas, there isn't any proof of inclusion in a certificate transparency log.
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-signing/verify-enforce-signed-images.md:36]: Would "Sigaas" make sense for customers? Should we use "Enforce Signing" or something else?

File: content/chainguard/chainguard-enforce/chainguard-enforce-signing/verify-enforce-signed-images.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      trustRootRef: my-sigstore-keys
+```
+
+This policy verifies that certain images satifying the glob pattern need to be signed using a specific trust root and a list of defined identities.
+
+In the following, we detail the purpose of the different fields:
+
+* `keyless.insecureIgnoreSCT` is enabled due to the lack of a certificate transparency log in the Enforce signing infrastructure.
+
+* `keyless.url` sets the url of the enforce sigstore instance used to sign the images. You can obtain the value by getting the `hostname` of your instace using the command `chainctl sigstore ca describe $ENFORCE_CA_NAME`. Replace `$ENFORCE_CA_NAME` with the name of your sigstore ca.
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-signing/verify-enforce-signed-images.md:47]: Just some casing changes here

```suggestion
* `keyless.url` sets the URL of the Enforce Sigstore instance used to sign the images. You can obtain the value by getting the `hostname` of your instace using the command `chainctl sigstore ca describe $ENFORCE_CA_NAME`. Replace `$ENFORCE_CA_NAME` with the name of your Sigstore CA.
```

**Review by ltagliaferri — APPROVED** [review]
I just have one question about the comment in the policy, referring to #sigaas

File: content/chainguard/chainguard-enforce/chainguard-enforce-signing/verify-enforce-signed-images.md (hunk 1 above)
Comment (hectorj2f) [reply content/chainguard/chainguard-enforce/chainguard-enforce-signing/verify-enforce-signed-images.md:36]: Yeah, we better use Enforce signing.

**Pull request merged**